dml_events.go

```diff
@@ -406,17 +410,21 @@ func Int64Value(value interface{}) (int64, bool) {
 //
 // ref: https://github.com/mysql/mysql-server/blob/mysql-5.7.5/mysys/charset.c#L963-L1038
 // ref: https://github.com/go-sql-driver/mysql/blob/9181e3a86a19bacd63e68d43ae8b7b36320d8092/utils.go#L717-L758
-func appendEscapedString(buffer []byte, value string) []byte {
+func appendEscapedString(buffer []byte, value string, padToLength uint) []byte {
```

Comment:
Can you rename this to `rightPadLengthForBinaryColumn` to make it more verbose? Can you also leave some comments in the code to link to this PR so people in the future can figure out why this is done?